administrator/components/com_tags/src/Service/HTML/AdministratorService.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_tags
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
```suggestion
 * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
```
unless I am mistaken this is a new file